pymepix/pymepix_connection.py

```diff
@@ -86,14 +88,23 @@ def data_thread(self):
             data_type, data = value
             self._event_callback(data_type, data)
 
+            self._channel.send_data_by_message_type(data_type, data)
+
 
             
 
     def __init__(self,
                  spidr_address=(cfg.default_cfg["timepix"]["tpx_ip"], 50000),
                  src_ip_port=('192.168.1.1', 8192), 
-                 pipeline_class=PixelPipeline):
+                 pipeline_class=PixelPipeline,
+                 chan_address=('127.0.0.1', 5056)):
```

Comment:
We should probably remove this hard-coded address, as well as we should remove it for `src_ip_port` and rather issue a warning or take if from the `default.yaml`.

Reply:
Done.